dockerfiles/run/spring/provisioned/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Run James
+#
+# VERSION	1.0
+
+FROM quynhnn/james-project-jpa
```

Comment:
Yes, it is and when this PR is validated and get approval, we have to push the base image to Linagora docker reporsitoris